src/main/java/object/piece/Piece.java

```diff
@@ -0,0 +1,63 @@
+package object.piece;
+
+import object.board.Board;
+import object.coordinate.Coordinate;
+import java.util.stream.Collectors;
+import object.team.Country;
+
+public class Piece {
+
+    private final Country country;
+    private final PieceType pieceType;
+
+    public Piece(Country country, PieceType pieceType) {
+        this.country = country;
+        this.pieceType = pieceType;
+    }
+
+    public boolean canMove(Board board, Coordinate departure, Coordinate arrival) {
+        if (!validateObstacle(board, departure, arrival)) {
+            return false;
+        }
+        if (!validateMovable(departure, arrival)) {
+            return false;
+        }
+        return true;
```

Comment:
아래처럼 줄여서 표현할 수 있겠네요!

```suggestion
        return validateObstacle(board, departure, arrival) && validateMovable(departure, arrival);
```

Reply:
저 방식이 가독성 측면에서 더 좋을 것 같네요. 수정했습니다!